src/test/java/com/wikia/webdriver/testcases/adstests/TestFloorAdhesion.java

```diff
@@ -0,0 +1,90 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.dataprovider.mobile.MobileAdsDataProvider;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.templates.TemplateDontLogout;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.mobile.MobileAdsBaseObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class TestFloorAdhesion extends TemplateDontLogout {
+    private final String FLOOR_ADHESION_SELECTOR = "#ext-wikia-adEngine-template-footer";
```

Comment:
Selector is "just" either XPath or CSS selector. CSS would mean the stylesheet or the CSS URL to me (before I think it can be also a CSS selector)